doc/user/content/sql/create-source/kafka.md

```diff
@@ -298,6 +298,15 @@ As long as any offset continues increasing, Materialize is consuming data from
 the upstream Kafka broker. For more details on monitoring source ingestion
 progress and debugging related issues, see [Troubleshooting](/ops/troubleshooting/).
 
+## Required permissions
+
+The access control lists (ACLs) on the Kafka cluster must permit Materialize
+to perform the following operations on the following resources:
+
+Operations  | Resource type    | Resource name
+------------|------------------|--------------
+Read        | Topic            | The specified `TOPIC` option
```

Comment:
The Kafka source will also attempt to commit offsets under the consumer group it creates, though IIRC it will fail open. We might want to include that permission here to avoid surprises later.

Reply:
Oh, great point, thanks!